.ci/validation.sh

```diff
@@ -1105,6 +1105,27 @@ ci-temp-check)
     exit $fail
   ;;
 
+  check-github-workflows-concurrency)
+    GITHUB_WORKFLOW_FILES=$(find .github/workflows -maxdepth 1 -not -type d -name "*.y*ml")
```

Comment:
> "*.y*ml"

Is this to include `yaml` files too? I don't see any changes in this PR for `yaml`.

Reply:
I do not see any "yaml" at https://github.com/checkstyle/checkstyle/tree/master/.github/workflows

Reply:
Yes, just future-proofing in case someone uses yaml extension.